pl_bolts/models/detection/detr/detr_utils.py

```diff
@@ -0,0 +1,115 @@
+import numpy as np
+import torch.nn as nn
+import torch
+from torchvision.ops.boxes import box_area
```

Comment:
this shall fo to try/catch

Reply:
Moved to try-catch. This file will get cleaned up with next torchvision release.
We would need only `accuracy` and `masks_to_boxes` (if needed).
This codebase has shifted to torchvision and I guess we can use it natively.

Reply:
> This file will get cleaned up with next torchvision release.
> This codebase has shifted to torchvision and I guess we can use it natively in october.

so you say that these metrics will be directly in pyTorch 1.7?

Reply:
Yes most should be.
I will patch it in October. It will lessen our code to test and break.